tempesta_fw/ss_skb.c

```diff
@@ -236,14 +236,11 @@ __it_next_data(struct sk_buff *skb, int i, TfwStr *it)
 static inline void
 __skb_insert_after(struct sk_buff *skb, struct sk_buff *nskb)
```

Comment:
I believe all our own list stuff like `ss_skb_queue_head_init()`, `ss_skb_queue_empty()`, `ss_skb_queue_tail()` and so on must be removed now.